src/main/java/pl/agh/edu/master_diet/service/ActivitySearchService.java

```diff
@@ -19,13 +19,12 @@
 public class ActivitySearchService extends SearchService<Activity> {
 
     private final ConversionService conversionService;
+    private final static Double METS_COEFFICIENT = 3.5;
```

Comment:
Why? Static and non static fields should be separated

Reply:
changed